src/tools/bin/nextstep_from_period.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017-2018 Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
```

Comment:
Please add [`set -euf -o pipefail`](https://github.com/azet/community_bash_style_guide#style-conventions) for modern bash scripts.
Then pls verify it still works and modernize if needed.